docs/source/bot_methods.rst

```diff
@@ -0,0 +1,291 @@
+.. raw:: html
+
+   <details>
+   <summary>Sending Messages</summary>
+
+.. list-table:: Sending Messages
+        :align: left
+        :widths: 1 4
+
+        * - :meth:`send_message`
+          - Used for sending text messages
+        * - :meth:`send_photo`
+          - Used for sending photos
+        * - :meth:`send_audio`
+          - Used for sending audio files
+        * - :meth:`send_document`
+          - Used for sending documents
+        * - :meth:`send_video`
+          - Used for sending videos
+        * - :meth:`send_animation`
+          - Used for sending animations
+        * - :meth:`send_voice`
+          - Used for sending voice messages
+        * - :meth:`send_video_note`
+          - Used for sending video notes
+        * - :meth:`send_media_group`
+          - Used for sending media grouped together
+        * - :meth:`send_location`
+          - Used for sending location
+        * - :meth:`send_venue`
+          - Used for sending venue locations.
+        * - :meth:`send_contact`
+          - Used for sending contacts
+        * - :meth:`send_poll`
+          - Used for sending polls
+        * - :meth:`send_sticker`
+          - Used for sending stickers
+        * - :meth:`send_invoice`
+          - Used for sending an invoice
+        * - :meth:`send_game`
+          - Used for sending a game
+        * - :meth:`send_dice`
+          - Used for sending dice messages
+        * - :meth:`send_chat_action`
+          - Used for sending chat actions
+        * - :meth:`forward_message`
+          - Used for forwarding messages
+        * - :meth:`copy_message`
+          - Used for forwarding messages without original sender name.
+
+.. raw:: html
+
+   </details>
+
+.. raw:: html
+
+   <details>
+   <summary>Updating Messages</summary>
+
+.. list-table:: Updating Messages
+    :align: left
+    :widths: 1 4
+
+    * - :meth:`edit_message_text`
```

Comment:
Are the methods in the sections ordered by some principle? maybe just order them alphabetically?

Reply:
yeah they are roughly. similar methods are grouped together. Like in the stickers section, we have `set_chat_sticker_set` and `delete_chat_sticker_set` next to each other. Still can make some alphabetical tho